tools/kmp-json-file.ts

```diff
@@ -0,0 +1,79 @@
+interface KmpJsonFile {
```

Comment:
Where will the official documentation (i.e., meaning of each field) for the JSON file exist?

Reply:
I have an open PR on the help site (currently private, although shortly hoping to move it public) which documents these fields. See https://help.keyman.com/developer/11.0/reference/file-types/metadata (probably will be in 12.0 URL when that lands).